src/sgx/rust_sgx_bridge/rust_sgx_bridge.cpp

```diff
@@ -62,6 +65,21 @@ int r_free_enclave(r_sgx_enclave_id_t *eid)
     return 0;
 }
 
+int r_get_epid_group_from_enclave(r_sgx_enclave_id_t *eid, r_sgx_epid_group_t *epid_group) {
```

Comment:
can you rename the function to "r_get_epid_group". EPID is not from Enclave

Reply:
Ok, will rename the function